.github/actions/build-dotnet/action.yml

```diff
@@ -33,6 +33,14 @@ inputs:
       Change this to any other value to alter the cache key, effectively invalidating the latest cache. This is the only
       current way to force "clear" the cache (https://github.community/t/how-to-clear-cache-in-github-actions/129038/5)
       until the associated issue (https://github.com/actions/cache/issues/2) is resolved.
+  publish-version:
+    required: false
+    default: "1.${{ github.run_number }}.${{ github.run_attempt }}-${{ github.run_id }}"
+    description: The value given to `dotnet build`'s `-Version` switch.
```

Comment:
Can this be markdown?

Reply:
I'm not aware that it's processed by anything so it can be whatever we want. I've started to use backtick in the commit descriptions too, becuase GH renders them as code so I think it's reasonable to use them here too.

Reply:
@DAud-IcI pointed me to this thread from another question, because a few occurrences of this: If this isn't rendered anywhere (as markdown), we're just looking at it as-is, so this means that we have a new way to emphasize something (code included) instead of double quotes. Double quotes work in descriptions and that's what we've been using before, IMHO backtick is a downgrade in terms of readability too.
If it was rendered, then we should use it like this in all descriptions, but we've got inconsistent formatting now.

Reply:
I don't see why backticks are less readable than double quotes, both are surrounding punctuation. Can you explain that?

We are all developers working on GitHub so we all know what backtick means in this context. Double quotes would be a downgrade semantically because they are less specific. We should use backtick to indicate code in all workflow/action descriptions.

Reply:
Double quotes in these descriptions only work well because YAML is liberal with how it handles strings (or should I say, inconsistent), since wrapping the full description in quotes will be understood as a string, but you can also omit the quotes, but if you add quotes in the middle of the description then the whole description will still be understood as one string... My point is, that backticks are fine for code, and if we can get away without surrounding quotes (like on line 38 above) then we should do that. Otherwise, use quotes in the descriptions according to English grammar.